boolean-6.js

```diff
@@ -0,0 +1,28 @@
+/* Americans are odd people: in their buildings, the first floor is actually the ground floor and there is no 13th floor (due to superstition).
+Write a function that given a floor in the american system returns the floor in the european system.
+With the 1st floor being replaced by the ground floor and the 13th floor being removed, the numbers move down to take their place. In case of above 13, they move down by two because there are two omitted numbers below them.
+Basements (negatives) stay the same as the universal level.
+
+Examples
+1  =>  0
+0  =>  0
+5  =>  4
+15  =>  13
+-3  =>  -3
+
+ */
+
+function getRealFloor(n) {
+    if (n === 0)
+        return (0)
+    if (n === -Math.abs(n))
+        return (-Math.abs(n))
```

Comment:
czemu nie po prostu n < 0, n === 0, n <= 13 i na koncu bez warunku dla n > 13?

Reply:
Używałem we wcześniejszych zadania do licz ujemnych Math.abs, i tak z rozpędu. Poprawiać?